Terraform/modules/ec2/ec2.tf

```diff
@@ -1,8 +1,10 @@
+# EC2 instances
```

Comment:
How to create 1 EC2 using your module?

Reply:
In this case the only option to create a new EC2 instance is to add new resource to the main block.
Alternatively EC2 module can be changed by adding a new resource, or the whole logic of the EC2 module should be changed.